build.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e # Stop script on error
+set -a # Automatically export all variables
+
+#install node
+PWD_NOW=$PWD
```

Comment:
There should be no necessity to install any application specific runtime on the machine itself. All of this should be containerized within Docker for ease of deployment and upgrade.

Reply:
You are right.